tests/indices/managed/test_google.py

```diff
@@ -11,19 +11,34 @@
 except ImportError:
     has_google = False
 
-from llama_index.indices.managed.google.generativeai import GoogleIndex
+from llama_index.indices.managed.google.generativeai import (
+    GoogleConfig,
+    GoogleIndex,
+    set_google_config,
+)
 
 SKIP_TEST_REASON = "Google GenerativeAI is not installed"
 
 
 if has_google:
     import llama_index.vector_stores.google.generativeai.genai_extension as genaix
 
-    genaix.set_defaults(
-        genaix.Config(api_endpoint="No-such-endpoint-to-prevent-hitting-real-backend")
+    genaix.set_config(
```

Comment:
Nit: Maybe use set_google_config and GoogleConfig instead?

Reply:
Done.

Reply:
undid this because of the imports issue :)